File: 1018.c

int main()
{
    int N, remain;
    scanf("%d",&N);
    printf("%d\n",N);

    printf("%d nota(s) de R$ 100,00\n",N/100);
    remain=N%100;

    printf("%d nota(s) de R$ 50,00\n",remain/50);
    remain=remain%50;

    printf("%d nota(s) de R$ 20,00\n",remain/20);
    remain=remain%20;

    printf("%d nota(s) de R$ 10,00\n",remain/10);
    remain=remain%10;

    printf("%d nota(s) de R$ 5,00\n",remain/5);
    remain=remain%5;

    printf("%d nota(s) de R$ 2,00\n",remain/2);
    remain=remain%2;

    printf("%d nota(s) de R$ 1,00\n",remain/1);
    return 0;

}

File: Python/1018.py
n=int(input())
print(n)
print(f"{int(n/100)} nota(s) de R$ 100,00")
r=n%100
print(f"{int(r/50)} nota(s) de R$ 50,00")
r=r%50
print(f"{int(r/20)} nota(s) de R$ 20,00")
r=r%20
print(f"{int(r/10)} nota(s) de R$ 10,00")
r=r%10
print(f"{int(r/5)} nota(s) de R$ 5,00")
r=r%5
print(f"{int(r/2)} nota(s) de R$ 2,00")
r=r%2
print(f"{int(r/1)} nota(s) de R$ 1,00")